### ISSUE TITLE
Cannot install integration (v2.0.1) via HACS

### ISSUE THREAD

**Issue by bigfeetneedbigboots** [And3rsL/Deebot-for-Home-Assistant#95]
**DESCRIPTION**
Cannot successfully install this integration (v2.0.1) via HACS. It creates /config/custom_components/deebot directory successfully and creates 5 files in this directory successfully however upon restart of Home Assistant (HA) the HACS integration is showing as available for installation (i.e. not installed).

**CONFIGURATION**
```
deebot:
  username: !secret deebot_username
  password: !secret deebot_password
  country: AU
  continent: WW
  deviceid:
    - !secret deebot_deviceid
```

**SCREENSHOTS**
<img width="1548" alt="Screen Shot 2021-03-23 at 11 18 28 am" src="https://user-images.githubusercontent.com/59955188/112075285-558faa80-8bcc-11eb-966d-4a1a7a8cdc40.png">

(Showing HACS on left (dark theme) in which this integration is not loaded whereas others are AND showing folders/files on right (light theme) showing Deebot directory created and files within)

**SYSTEMS**
 - Device: Ecovacs Deebot Ozmo 900 Series Model DN5G
 - Home Assistant Version: core-2021.3.4 / supervisor-2021.03.6
 - Hardware: NUC 10
 - Home Assistant Installation: Home Assistant Operating System (HassOS)

**CONTEXT**
I have been using the Ligio hacc-ozmo integration (manually installed) successfully but wanted to move to this one because it is delivered via HACS and I like the live map features this integration appears to offer (like I get in the Ecovacs App I presume).
I completely removed the Ligio hacc-ozmo configuration and deleted all of the files and restarted before trying to install this integration. 
I have tried the delete all files and restart 6-7 times without any success. I even restarted the host and reinstalled without success. I've tried various combinations of leaving the configuration there and commenting it out. Nothing seems to work. It is almost like HA refuses to acknowledge the installation of this integration.

**LOGS**
There are no logs relating to this integration.

**Comment by And3rsL** [comment]
Hi, unfortunately if your robot is working with ligio integration then this integration is not compatible with your robot

**Comment by bigfeetneedbigboots** [comment]
Why is that? The configuration required for that integration is almost identical to this one and my robot is an Ozmo. In any case, the bug is that I can’t get it to install the integration at all from HACS. The type of robot is irrelevant at this point of the procedure.

**Comment by And3rsL** [comment]
> Why is that? The configuration required for that integration is almost identical to this one and my robot is an Ozmo.

Ecovacs robots use different API/technologies across their robots, i have decided to support model like 'Deebot 950' since is the only one that i own and i can't test others
The configuration is almost identical because their login system is the same across all devices, the differences become evident when you try to comunicate with the robot

> In any case, the bug is that I can’t get it to install the integration at all from HACS. The type of robot is irrelevant at this point of the procedure.

HACS just clones files in custom_components, since all files needed are there you can ignore the fact that its saying that is not installed, anyway i can't help you about it, you should open an issue in HACS repository

**Comment by filippolmt** [comment]
Hi, I also have the same problem as you, unfortunately I can't install it, I restart the home assistant but then the repository returns with the new tag

**Comment by msfuture** [comment]
Same problem in my setup. Install, reboot and not loaded. Instead it is marked as new repository ready to install. Folders are created, but seem to be ignored.

**Comment by Yterz** [comment]
Same, but I had 2.0.1 already instal since release, and now HACS doesn't acknowledge that anymore, and push for an installation as a new repository.
Anyway, HACS still properly install it in config/custom_components/deebot, and it still work after a re-instal, nothing changed. But it seems it doesn't make the link with this Github anymore...

Quick difference I found, in 2.0.1 HACS's manifest.json there is
```"documentation" : "https://github.com/And3rsL/Deebot-for-hassio"```
while when I download directly the 2.0.1 master from this Github in manifest.json there is 
```"documentation": "https://github.com/And3rsL/Deebot-for-Home-Assistant"```

Was this Github url changed recently ? It seems HACS didn't got this information properly or something...

**Comment by bigfeetneedbigboots** [comment]
@And3rsL given that others are seeing the same problem with installing this integration via HACS, how about taking a look? Yterz has given some detailed information in their post above about differences between versions. Did something change between recent versions which is causing HACS not to pick it up anymore?

I'll lodge an issue on the HACS repository if you really want but I think they will dismiss it saying that other integrations work fine (including all those on my installation) and that I should raise an issue with the problematic integration (i.e. yours).

**Comment by And3rsL** [comment]
> Same, but I had 2.0.1 already instal since release, and now HACS doesn't acknowledge that anymore, and push for an installation as a new repository.
> Anyway, HACS still properly install it in config/custom_components/deebot, and it still work after a re-instal, nothing changed. But it seems it doesn't make the link with this Github anymore...
> 
> Quick difference I found, in 2.0.1 HACS's manifest.json there is
> ```"documentation" : "https://github.com/And3rsL/Deebot-for-hassio"```
> while when I download directly the 2.0.1 master from this Github in manifest.json there is 
> ```"documentation": "https://github.com/And3rsL/Deebot-for-Home-Assistant"```
> 
> Was this Github url changed recently ? It seems HACS didn't got this information properly or something...
> 
> (Device: Deebot T8 AIVI+ ; Home Assistant 2021.3.4 ; Hardware: Raspberry Pi 4 ; Deebot-for-hassio : 2.0.1)

Yes, name changed. I will rename it again and when the new version is released i will fix it :)

**Comment by bigfeetneedbigboots** [comment]
Thanks @And3rsL. Happy to help with testing it when you release the new version.

**Comment by Yterz** [comment]
The HACS issue is fixed with 2.1.1 for me

**Comment by edenhaus** [comment]
I will close it as it seem to be resolved
